docs/changelog.md

```diff
@@ -17,6 +17,16 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/).
 
 ### Changed
 
+- **Breaking**: Temporarily remove React support to unblock dependency updates.
+
+  React no longer works with the latest Stencil output target, which is preventing us from updating
+  our dependencies to their latest versions. Since the team currently lacks the resources to fix
+  this issue, we're removing React support temporarily to prevent it from hindering progress.
+
+- Updated dependencies and cleand up test config for Stencil.
+
+- Fixed mermaid diagrams in component documentation.
```

Comment:
Here I would provide a Fixed section, not sure that fixing the mermaid diagrams under the changed section is the right way to go